Road/source/data/get_data.py

```diff
@@ -0,0 +1,21 @@
+from telethon import TelegramClient,events
+import csv
+
+api_id = 28394694 
+api_hash = '4c0b02c61f2626bd3f0d1b58883ec759'
```

Comment:
Isn't it better to put these variables in constants.py file?

Reply:
this data should be shared internally and we put them in a  .env file with extra code lines we make them env variables, but we don't need them here when we write the endpoint that will read the messages in real-time we do that thing 
ps: you cannot use this information alone to get messages you need my phone to get a PIN when you start the app , so its not a secret informations

Reply:
Thanks for clarifying.